src/mca/schizo/ompi/help-schizo-ompi.txt

```diff
@@ -36,30 +36,29 @@ Initiate an instance of the PMIx Reference RTE (PRRTE) DVM
    --debug-daemons-file              Enable debugging of any PRTE daemons used by this application, storing
                                      their verbose output in files
    --display <arg0>                  Comma-delimited list of options for displaying information about the
-                                     allocation and job. Allowed values: allocation, bind, map, map-devel,
-                                     topo
+                                     allocation and job. Allowed values for <arg0>: allocation, bind,
+                                     map, map-devel, topo
    --get-stack-traces                Get stack traces of all application procs on timeout
    --leave-session-attached          Do not discard stdout/stderr of remote PRTE daemons
    --report-state-on-timeout         Report all job and process states upon timeout
    --spawn-timeout <arg0>            Timeout the job if spawn takes more than the specified number of seconds
-   --stop-on-exec                    If supported, stop each specified process at start of execution
-   --stop-in-init                    Direct the specified processes to stop in PMIx_Init
-   --stop-in-app                     Direct the specified processes to stop at an application-controlled location
    --test-suicide <arg0>             Suicide instead of clean abort after delay
    --timeout <arg0>                  Timeout the job after the specified number of seconds
-   --output-proctable <arg0>         Print the complete proctable to stdout [-], stderr [+], or a file
-                                     [anything else] after launch
+   --output-proctable                Print the complete proctable to stdout after launch
```

Comment:
My bad - my instructions were out of date :(. Can you put back:

`stdout [-], stderr [+], or a file  [anything else] after launch1`. @rhc54 fixed this so it would accept these.

I believe this is also an optional argument, so an `=` will need to be used. (default is +)